config/sidekiq_scheduler.yml

```diff
@@ -130,6 +130,11 @@ VBADocuments::ReportUnsuccessfulSubmissions:
   class: VBADocuments::ReportUnsuccessfulSubmissions
   description: "Weekly report of unsuccessful submissions"
 
+VBADocuments::RunUnsuccessfulSubmissions:
+  cron: "0 0 * * MON-FRI America/New_York"
```

Comment:
Is this correct for running the job every day? @bastosmichael

Reply:
Looks good but make sure that the RunUnsuccessfulSubmissions gets triggered before ReportUnsuccessfulSubmissions time wise.

Reply:
@bastosmichael updated to run daily at 2300 hours.